src/Kdyby/Annotations/DI/AnnotationsExtension.php

```diff
@@ -81,7 +82,7 @@ public function getConfig(array $defaults = NULL, $expand = TRUE)
 			$config = Nette\DI\Config\Helpers::merge($config, ['ignore' => $globalConfig['doctrine']['ignoredAnnotations']]);
 		}
 
-		return $this->compiler->getContainerBuilder()->expand($config);
+		return Nette\DI\Helpers::expand($config, $this->getContainerBuilder()->parameters);
```

Comment:
Not to apply it, but for me to understand it. I have no idea, what you mean.

I thought expand is used for `%appDir%` => absolute path conversion.